zerver/views/messages.py

```diff
@@ -1359,12 +1359,20 @@ def update_message_backend(request: HttpRequest, user_profile: UserMessage,
                            message_id: int=REQ(converter=to_non_negative_int),
                            topic_name: Optional[str]=REQ_topic(),
                            propagate_mode: Optional[str]=REQ(default="change_one"),
-                           content: Optional[str]=REQ(default=None)) -> HttpResponse:
+                           content: Optional[str]=REQ(default=None),
+                           client_oldest_message_id: Optional[int]=REQ(default=None)
+                           ) -> HttpResponse:
 
     if not user_profile.realm.allow_message_editing:
         return json_error(_("Your organization has turned off message editing"))
 
     message, ignored_user_message = access_message(user_profile, message_id)
+
+    # Oldest message of the topic which is visible to the client.
+    client_oldest_message = None
+    if client_oldest_message_id:
+        client_oldest_message, ignored_client_oldest_user_message = access_message(user_profile,
```

Comment:
this second variable can be `ignored_user_message`; no need for a separate variable name for a value we're ignoring.